logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/ShouldAskCallFeedbackUseCase.kt

```diff
@@ -24,23 +25,59 @@ import com.wire.kalium.util.DateTimeUtil
 import kotlinx.datetime.Instant
 
 /**
- * Use case that returns [Boolean] if user should be asked for a feedback about call quality or not.
+ * Use case to determine if the call feedback should be asked.
  */
 interface ShouldAskCallFeedbackUseCase {
-    /**
-     * @return [Boolean] if user should be asked for a feedback about call quality or not.
-     */
-    suspend operator fun invoke(): Boolean
+    suspend operator fun invoke(
+        establishedTime: String?,
+        currentTime: String = DateTimeUtil.currentIsoDateTimeString()
+    ): ShouldAskCallFeedbackUseCaseResult
 }
 
-@Suppress("FunctionNaming")
-internal fun ShouldAskCallFeedbackUseCase(
+internal fun shouldAskCallFeedbackUseCase(
```

Comment:
same question here

Reply:
I guess I just missed the moment when we started doing it that way in use cases 😅 
I agree, reverting it to uppercase would be better